package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "webpack.config.js",
+  "scripts": {
```

Comment:
- Please set up Webpack development server, please add basic commands for building Webpack in development mode, currently, there are errors in the terminal when starting your application  👍🏿